biztalk/dev-center/learn-biztalk-server.md

```diff
@@ -25,11 +25,11 @@ Opens QuickLearn.com.
 
 ## Channel 9 videos
```

Comment:
Channel 9 is now "Microsoft Learn", but all these listed videos are now gone, so delete this section.